Test/Resolve-WhiskeyTaskPathParameter.Tests.ps1

```diff
@@ -60,7 +104,7 @@ function ThenTaskCalled
         $taskParameters.Count | Should -Be $WithParameter.Count
         foreach( $key in $WithParameter.Keys )
         {
-            $taskParameters[$key] | Should -Be $WithParameter[$key] -Because $key
+            $taskParameters[$key] | Sort-Object | Should -Be ($WithParameter[$key] | Sort-Object) -Because $key

```

Comment:
I might add (useless) parenthesis around first .. | sort for visual parity with -Be.